packages/AutoTDD-GUI.package/ATDDBrowser.class/instance/buildButtonPerforming.withLabel.with..st

```diff
@@ -1,5 +1,5 @@
 building
-buildButtonWithMethod: aMethod withLabel: aString with: aBuilder
+buildButtonPerforming: aMethod withLabel: aString with: aBuilder
```

Comment:
Haven't seen the term "performing" in squeak yet. Maybe we can use "do"?

```suggestion
buildButtonWith: aBuilder label: aString do: aMethod
```

Reply:
Good suggestion!